.rhcicd/clowdapp-aggregator.yaml

```diff
@@ -61,6 +61,8 @@ objects:
           value: ${ENV_NAME}
         - name: PROMETHEUS_PUSHGATEWAY_URL
           value: ${PROMETHEUS_PUSHGATEWAY}
+        - name: NOTIFICATIONS_AGGREGATOR_ACCORDING_ORG_PREF
```

Comment:
Comment about the entire `cronjob` object in this file: I think we'll need to duplicate that `cronjob`. The old one will remain unchanged while the new one will have a different schedule and be disabled by default from the env vars / app-interface configuration. Then, when we'll be ready to switch from the old to the new cronjob, we'll do that from app-interface and we'll enable the feature flag at the same time.